ProcessLib/BoundaryCondition/BoundaryCondition.cpp

```diff
@@ -176,6 +183,18 @@ BoundaryConditionBuilder::createRobinBoundaryCondition(
 }
 
 std::unique_ptr<BoundaryCondition>
+BoundaryConditionBuilder::createNonuniformDirichletBoundaryCondition(
+    const BoundaryConditionConfig& config,
+    const NumLib::LocalToGlobalIndexMap& dof_table, const MeshLib::Mesh& mesh,
+    const int variable_id, const unsigned integration_order,
+    const unsigned shapefunction_order)
```

Comment:
Are integration and shape fct order needed?

Reply:
Maybe leave them until the next refactoring.

Reply:
Refactoring taking place now ;) (A small one.)